### ISSUE TITLE
removing some default image sizes won't work

### ISSUE THREAD

**Issue by tomceek** [fredrikekelund/grav-plugin-resize-images#11]
![capture-resize](https://user-images.githubusercontent.com/30144585/30175648-f98696e6-9407-11e7-95a6-995ab0a9256d.PNG)
When I delete some unnecessary image sizes after clearing cache they always return.

-----
The main problem with this plugin is that if I have 15 or 20 images in media, it won't handle nicely without noticing that memory limit on shared hosting is exceeding. So plugin should work differently while resizing images or at least let me to remove unnecessary image sizes.

**Comment by fredrikekelund** [comment]
Thanks for reporting this, @tomceek! I'll try and have a look at this in a couple of days to see if I can reproduce the behaviour

**Comment by tomceek** [comment]
any updates?

**Comment by fredrikekelund** [comment]
@tomceek no, sorry. I would encourage you to send a PR, though! It'll probably be faster than me looking into it, which might still take a while
